modules/nf-core/modules/ratioconv/main.nf

```diff
@@ -0,0 +1,34 @@
+// Import generic module functions
```

Comment:
This module should be moved to the `modules/local` directory as it is not in the nf-core/modules repository